elks/fs/minix/dir.c

```diff
@@ -89,11 +89,11 @@ static int minix_readdir(struct inode *inode,
 	do {
 	    de = (struct minix_dir_entry *) (offset + bh->b_data);
 	    if (de->inode) {
-		if (filldir(dirent, de->name, strnlen(de->name, info->s_namelen),
-			filp->f_pos, de->inode) < 0) {
+			filldir(dirent, de->name, strnlen(de->name, info->s_namelen),
```

Comment:
Indentation level wrong?

Reply:
Actually not in the new code (using only TABs to indent the beginning of the lines). But as the old code mixes spaces and TABs to indent the beginning of the lines, it depends on the TAB rendering. And I avoid to touch the old code to clearly identify the real changes.